CurrencyConverter.playground/Contents.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+var currencyFormatter: NumberFormatter = {
+    let formatter = NumberFormatter()
+    formatter.numberStyle = .currency
+    return formatter
+}()
+
+enum Currency {
+    case cad
+    case mxn
+}
+let usdToCad = 1.41
+let usdToMxn = 23.92
+var currency: Currency = .cad
+
+
+func convert(_ dollars: Double) -> Double {
+    
+    var convertedDollars = 1.0
+
+    switch currency {
+        case .cad:
+            convertedDollars =  dollars * usdToCad
+
+        case .mxn:
+            convertedDollars =  dollars * usdToMxn
+    }
+    _ = currencyFormatter.string(from: convertedDollars as NSNumber)
+
+    return Double(convertedDollars)
+
+
+}
+
+func convert(amountString: String) -> String? {
+    
+    guard let amount = Double(amountString) else {
+        return ""
+    }
+    let cad = convert(amount)
+        return String(cad)
```

Comment:
Maybe name this variable something that is agnostic to the currency being converted to, such as `convertedAmount`. This way you can switch the currency between mxn and cad or any other currency that you would add in the future.